src/openrct2-ui/ride/Construction.cpp

```diff
@@ -24,6 +24,169 @@ using namespace OpenRCT2::TrackMetaData;
 
 namespace OpenRCT2
 {
+    /**
+     * Order of special track elements dropdown. Elements with the same name string must be sequential or they show up twice.
+     */
+    constexpr std::array kSpecialElementsDropdownOrder = {
+        TrackElemType::EndStation,
+        TrackElemType::SBendLeft,
+        TrackElemType::SBendRight,
+
+        // Loops
+        TrackElemType::LeftVerticalLoop,
+        TrackElemType::RightVerticalLoop,
+        TrackElemType::HalfLoopUp,
+        TrackElemType::HalfLoopDown,
+        TrackElemType::LeftLargeHalfLoopUp,
+        TrackElemType::LeftLargeHalfLoopDown,
+        TrackElemType::RightLargeHalfLoopUp,
+        TrackElemType::RightLargeHalfLoopDown,
+        TrackElemType::FlyerHalfLoopUninvertedUp,
+        TrackElemType::FlyerHalfLoopInvertedDown,
+        TrackElemType::FlyerHalfLoopInvertedUp,
+        TrackElemType::FlyerHalfLoopUninvertedDown,
+        TrackElemType::MultiDimInvertedFlatToDown90QuarterLoop,
+        TrackElemType::Up90ToInvertedFlatQuarterLoop,
+        TrackElemType::InvertedFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimUp90ToInvertedFlatQuarterLoop,
+        TrackElemType::MultiDimFlatToDown90QuarterLoop,
+        TrackElemType::MultiDimInvertedUp90ToFlatQuarterLoop,
```

Comment:
I suggest putting quarter loops at the end of the loops section, i.e. small->medium->big->quarter